1/calculator.rb

```diff
@@ -1,4 +1,33 @@
+#Constants defined
+
+ONE = 1
+TWO = 2
+THREE = 3
+FOUR = 4
+FIVE = 5
+SIX = 6
+SEVEN = 7
+EIGHT = 8
+NINE = 9
+
+#Functions defined
+
+def add(number=0,number2=0, number3=0,number4=0)
+  number + number2 + number3 + number4
+end
+
+def subtract(number=0,number_main=0)
```

Comment:
What was your thinking in using the name `number_main`? I like it, it distinguishes it from the other number.

Reply:
I had used `number_main` for the final number in subtraction which the other numbers subtracted from. Made sense to define it as the 'main number from which other numbers then interact with', which was also useful for division.